packages/next/server/image-optimizer.ts

```diff
@@ -352,7 +352,9 @@ export async function imageOptimizer(
           transformer.resize(width)
         }
 
-        if (contentType === WEBP) {
+        if (contentType === AVIF) {
+          transformer.avif({ quality })
+        } else if (contentType === WEBP) {
```

Comment:
you've just added the support for the case when `sharp` exists in deps, but what about the `squoosh` fallback? 
in the official [squoosh repository](https://github.com/GoogleChromeLabs/squoosh), I do see that [AVIF is supported](https://github.com/GoogleChromeLabs/squoosh/blob/dev/libsquoosh/src/codecs.ts) ;) should we bring it to Next repo? @styfle @timneutkens  wdyt?

Reply:
Yes it will need to be in both. I'm going to update squoosh soon after we get a few bugs fixed